01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,11 @@
+(1..20) .each do |x|
```

Comment:
```suggestion
(1..20).each do |n|
```

不要なスペースが空いてます。
あと、数値が入る変数はintegerのiやnumberのnを使った方がわかりやすいです